### PR TITLE
Enrich expense to add gender details to Payee

### PR DESCRIPTION
**Pull request by ajay-egov**
@shailesh-egov @jagankumar-egov

### CODE REVIEW

File: backend/expense/src/main/java/org/egov/digit/expense/web/models/IndividualSearch.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package org.egov.digit.expense.web.models;
```
Comment (shailesh-egov) [inline backend/expense/src/main/java/org/egov/digit/expense/web/models/IndividualSearch.java:1]: @ajay-egov we can not copy IndividualSearch model here. use Map to create these.

File: backend/expense/src/main/java/org/egov/digit/expense/web/models/IndividualSearchRequest.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package org.egov.digit.expense.web.models;
```
Comment (shailesh-egov) [inline backend/expense/src/main/java/org/egov/digit/expense/web/models/IndividualSearchRequest.java:1]: @ajay-egov we can not copy IndividualSearchRequest model here. use Map to create these.

File: backend/muster-roll/src/main/java/org/egov/util/MusterRollServiceUtil.java [hunk 3]
```diff
@@ -100,6 +100,7 @@ public void populateAdditionalDetails(Object mdmsData, IndividualEntry individua
 		additionalDetails.put("userName", matchedIndividual.getName().getGivenName());
 		additionalDetails.put("fatherName", matchedIndividual.getFatherName());
 		additionalDetails.put("mobileNo", matchedIndividual.getMobileNumber());
+		additionalDetails.put("gender",matchedIndividual.getGender());
```
Comment (shailesh-egov) [inline backend/muster-roll/src/main/java/org/egov/util/MusterRollServiceUtil.java:103]: @ajay-egov this is muster roll service move this into another PR.

File: backend/expense/src/main/java/org/egov/digit/expense/util/EnrichmentUtil.java [hunk 4]
```diff
@@ -66,7 +71,12 @@ public BillRequest encrichBillForCreate(BillRequest billRequest) {
             billDetail.getPayee().setParentId(billDetail.getBillId());
             billDetail.getPayee().setAuditDetails(audit);
             billDetail.getPayee().setStatus(Status.ACTIVE);
-            
+
+            List<String> Ids = new ArrayList<>();
+            Ids.add(billDetail.getPayee().getIdentifier());
+            Object additionalDetails = genderUtil.getGenderDetails(billRequest.getRequestInfo(),billDetail.getPayee().getTenantId(),Ids);
+            billDetail.getPayee().setAdditionalDetails(additionalDetails);
```
Comment (shailesh-egov) [inline backend/expense/src/main/java/org/egov/digit/expense/util/EnrichmentUtil.java:78]: @ajay-egov you are replacing the data of additional details without maintaining the existing available data.

**Review by shailesh-egov — CHANGES_REQUESTED** [review]

**Review by shailesh-egov — APPROVED** [review]

**Review by subhashini-egov — APPROVED** [review]
Commenting basis approval from Shailesh.

**Pull request merged**